<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetModel7</class>
 <widget class="QWidget" name="WidgetModel7">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1224</width>
    <height>724</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelImg">
         <property name="minimumSize">
          <size>
           <width>900</width>
           <height>600</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>900</width>
           <height>600</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="VideoWidget" name="widget" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="XyProcessLog" name="widgetLog" native="true">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="XyResultLabel" name="labelTips">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>点击检测设备环境是否正常</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">videowidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XyResultLabel</class>
   <extends>QLabel</extends>
   <header>XyResultLabel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>XyProcessLog</class>
   <extends>QWidget</extends>
   <header location="global">XyProcessLog.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
